src/environments/environment.prod.ts

```diff
@@ -70,7 +70,7 @@ export const environment = {
   RBSTest: `RBS Test`,
   visualAcuityTest: `Visual Acuity Test`,
   haemoglobinTest: `Hemoglobin Test`,
-  abhaExtension: `@abdm`,
+  abhaExtension: `@sbx`,
```

Comment:
@helenKaryamsetty is this added to CI template?
Can we use ABHA_MODE variable we added in pipeline yesterday?